packages/neutrino-preset-node/index.js

```diff
@@ -35,11 +35,13 @@ module.exports = (neutrino) => {
   neutrino.use(compile, {
     include: [neutrino.options.source, neutrino.options.tests],
     babel: {
-      plugins: [require.resolve('babel-plugin-dynamic-import-node')],
+      plugins: [[require.resolve('fast-async'), { spec: true }], require.resolve('babel-plugin-dynamic-import-node')],
       presets: [
         [require.resolve('babel-preset-env'), {
           modules: false,
-          targets: neutrino.options.compile.targets
+          useBuiltIns: false,
```

Comment:
`useBuiltIns` is on there because it's generating a library, while this preset is for application code. Having `useBuiltIns` will let babel-preset-env reduce babel-polyfill.